wiki/Featured_Artists/Featured_Artist_Showcase_Beatmaps/en.md

```diff
@@ -0,0 +1,359 @@
+  - (![][o!s]) [Hige Driver - Palette](https://osu.ppy.sh/beatmapsets/682416#osu/1443057) hosted by ![][flag_RU] [Jounzan](https://osu.ppy.sh/users/1848318)
+  - (![][o!s]) [Miracle Sugite Yabai (feat. shully)](https://osu.ppy.sh/beatmapsets/682290#osu/1442773) hosted by ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994)
+  - (![][o!s]) [Yuyoyuppe - Emerald Galaxy](https://osu.ppy.sh/beatmapsets/682286#osu/1442757) hosted by ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418)
+
+## 2018
+
+### May 2018
+
+- **[Cranky](https://osu.ppy.sh/home/news/2018-05-25-new-featured-artist-cranky)** (2018-05-25)
+  - (![][o!s]) [Cranky - Chandelier - King](https://osu.ppy.sh/beatmapsets/751785#osu/1582621) hosted by ![][flag_DE] [Mao](https://osu.ppy.sh/users/2204515)
+  - (![][o!s]) [Cranky - CHASER](https://osu.ppy.sh/beatmapsets/751972#osu/1583059) hosted by ![][flag_RU] [Jounzan](https://osu.ppy.sh/users/1848318)
+  - (![][o!s]) [Cranky - Feline, the White...](https://osu.ppy.sh/beatmapsets/751773#osu/1632945) hosted by ![][flag_NL] [appleeaterx](https://osu.ppy.sh/users/2407160)
+  - (![][o!s]) [Cranky - Hanaarashi](https://osu.ppy.sh/beatmapsets/751771#osu/1582579) hosted by ![][flag_RU] [Mirash](https://osu.ppy.sh/users/2841009)
+  - (![][o!s]) [Cranky - La fuite des jours](https://osu.ppy.sh/beatmapsets/751932#osu/1582988) hosted by ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994)
+  - (![][o!s]) [Cranky - Ran](https://osu.ppy.sh/beatmapsets/751772#osu/1582585) hosted by ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418)
+  - (![][o!s]) [Cranky - Rave 2 Rave](https://osu.ppy.sh/beatmapsets/751896#osu/1582902) hosted by ![][flag_KR] [Sonnyc](https://osu.ppy.sh/users/11771)
+  - (![][o!s]) [Cranky - T&J](https://osu.ppy.sh/beatmapsets/752022#osu/1583147) hosted by ![][flag_SG] [handsome](https://osu.ppy.sh/users/2123087)
+
+### August 2018
+
+- **[High Tea Music](https://osu.ppy.sh/home/news/2018-08-13-new-featured-artist-high-tea-music)** (2018-08-13)
+  - ![][o!s] [Andromedik - Invasion](https://osu.ppy.sh/beatmapsets/789544#osu/1656911) hosted by ![][flag_RU] [Mirash](https://osu.ppy.sh/users/2841009)
+  - ![][o!s] [Gourski x Himmes - Silence](https://osu.ppy.sh/beatmapsets/789905#osu/1657683) hosted by ![][flag_FR] [BOUYAAA](https://osu.ppy.sh/users/405449)
+  - ![][o!s] [James Portland - Sky](https://osu.ppy.sh/beatmapsets/789528#osu/1656874) hosted by ![][flag_SG] [handsome](https://osu.ppy.sh/users/2123087)
+  - ![][o!s] [Lexurus - Gravity](https://osu.ppy.sh/beatmapsets/789529#osu/1656879) hosted by ![][flag_CL] [Milan-](https://osu.ppy.sh/users/1052994)
+  - ![][o!s] [Phonetic - Journey](https://osu.ppy.sh/beatmapsets/795432#osu/1670491) hosted by ![][flag_KR] [Left](https://osu.ppy.sh/users/5062061)
+  - ![][o!s] [Pulse - LP](https://osu.ppy.sh/beatmapsets/789374#osu/1656604) hosted by ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418)
```

Comment:
these are mising the parentheses around the game mode icon. personally i think they look better without parentheses tho.